src/persist/src/nemesis/direct.rs

```diff
@@ -613,6 +616,37 @@ mod tests {
         nemesis::run(10, GeneratorConfig::default(), direct);
     }
 
+    impl StartRuntime for S3BlobConfig {
+        fn start_runtime(&mut self, unreliable: UnreliableHandle) -> Result<RuntimeClient, Error> {
+            let runtime = Arc::new(AsyncRuntime::new().expect("failed to create async runtime"));
+            let guard = runtime.enter();
+            let config = self.clone();
+            let blob = S3Blob::open_exclusive(config, ("reentrance0", "direct_file").into())?;
```

Comment:
nit: should "direct_file" be "direct_s3"?

Reply:
done